source/queue_policy/policies.rst

```diff
@@ -713,12 +715,12 @@ Jet and Hera
        combinations of the max running jobs per project-account that
        are permitted: 1. A project can have two 2-node jobs at 24
        hours of wallclock and one 1-node job at 72 hours of wallclock.
-       2. A project can have one 1-node job at 168 hours of wallclock.
+       1. A project can have one 1-node job at 168 hours of wallclock.
        Lowest priority QOS for use with GPU nodes. If you have an
        allocation of “windfall only” (Monthly allocation = 1) you can
        only submit to this QOS. Submitting to this QOS will NOT affect
        your future job priority FairShare Factor (f). EffectvUsage =
-       0. See How FairShare Works. This QOS is useful for low priority
+       1. See How FairShare Works. This QOS is useful for low priority
```

Comment:
I'm pretty sure this should have remained as a zero.  ``EffectvUsage = 0`` .  But this whole paragraph seems ... confusing.

Reply:
corrected; and can explain as well.